pulumitest/asserthelpers/helpers.go

```diff
@@ -0,0 +1,82 @@
+package asserthelpers
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/pulumi/pulumi/sdk/v3/go/common/apitype"
+)
+
+func FindUnexpectedOps(changeSummary map[string]int, expectedOpTypes ...apitype.OpType) []string {
```

Comment:
Well this is interesting looks like Automation API exposes change results. Which is great! And we have a model for it in apitype.OpType

https://github.com/pulumi/pulumi/blob/master/sdk/go/common/apitype/history.go#L63

This is great. 

Can we hide the "string" from public API here then and use apitype.OpType everywhere?